server/datastore/mysql/migrations/tables/20191220130734_AddLiveQueryDisabledToAppConfig.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up20191220130734, Down20191220130734)
+}
+
+func Up20191220130734(tx *sql.Tx) error {
+	_, err := tx.Exec(
+		"ALTER TABLE `app_configs` " +
+			"ADD COLUMN `live_query_disabled` TINYINT(1) NOT NULL DEFAULT FALSE;",
```

Comment:
I suggest using a boolean here to improve readability.

Reply:
I was being consistent with the existing code base in that every other setting flag uses TINYINT(1) already.